### PR TITLE
Reut/feat/routes

### PR DESCRIPTION
**Pull request by ReaGol**
Deleted Celebrate since it does not work well with mongoose, have to find solution to that with Arel and Eric on Sunday.
Joi validation is fine, although problem with string validation in director.
Winston works, Morgan also works.

### CODE REVIEW

File: components/movies/constants/movies.constants.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const defaultPoster =
+	'https://images.unsplash.com/photo-1616530940355-351fabd9524b?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8M3x8bW92aWUlMjBwb3N0ZXJ8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60';
+
+export const defaultActorPoster =
+	'https://images.unsplash.com/photo-1519058082700-08a0b56da9b4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTR8fHBvcnRyYWl0JTIwbWFufGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60';
```
Comment (AmeerLalaDeveloper) [inline components/movies/constants/movies.constants.js:5]: u dont need to create folder for constants , the path should be components/movies/YourFile

File: services/DB/mongoose.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import mongoose from "mongoose";
-import dotenv from 'dotenv'
-dotenv.config()
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
```
Comment (AmeerLalaDeveloper) [inline services/DB/mongoose.js:2]: remove this from here u have it already in index.js

File: services/DB/mongoose.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+dotenv.config();
```
Comment (AmeerLalaDeveloper) [inline services/DB/mongoose.js:3]: remove this from here u have it already in index.js

File: index.js [hunk 4]
```diff
@@ -1,11 +1,16 @@
 import cors from 'cors';
 import express from 'express';
+import morgan from 'morgan';
+import logger from './services/logger/index.js';
 import router from './components/movies/movie.routes.js';
```
[the remark this answers is not in the document]
Comment (ReaGol) [reply index.js]: Done

**Pull request merged**